example-crud/src/actions/index.js

```diff
@@ -29,11 +33,11 @@ const logout = async history => {
 
 const creatUserDocument = async userData => {
```

Comment:
Could we add to the README some basic instructions for creating the correct collections and structure with the new project.